### ISSUE TITLE
multi-entry plugin no such file error

### ISSUE THREAD

**Issue by aminya** [rollup/plugins#488]
- Rollup Plugin Name: < multi-entry-->
- Rollup Plugin Version: 3.0.1
- Rollup Version: 2.20.0
- Operating System (or Browser): Win 10
- Node Version: 14.5.0

### How Do We Reproduce?

In REPL.it https://repl.it/@aminya/x-terminal#rollup.config.js
Repository: https://github.com/aminya/x-terminal/tree/multientry

~~I could not reproduce it completely since the node version is 12. You need have 14.5.0, but here is the configuration:
https://repl.it/join/hgnuaflr-aminya~~

### Expected Behavior
```
src/x-a.js, src/x-b.js → dist...
[!] Error: ENOENT: no such file or directory, open 'C:\project\x-a\dist\_rollup:plugin-multi-entry:entry-point.js.map'
Error: ENOENT: no such file or directory, open 'C:\prject\x-a\dist\_rollup:plugin-multi-entry:entry-point.js.map'
```
while dist folder exists

### Actual Behavior

Bundling.

**Comment by shellscape** [comment]
Unfortunately we'll need a complete reproduction to triage this for you. I'd suggest making a repo with instructions for which node version to use in the README

**Comment by aminya** [comment]
I can set up my real-world repository so you can try.

**Comment by aminya** [comment]
@shellscape Here is the branch for you to try: https://github.com/aminya/x-terminal/tree/multientry

**Comment by shellscape** [comment]
That's appreciated, but it doesn't fit the definition of minimal as mentioned in the issue template. 

>   2. Provide a minimal repository link (Read https://git.io/fNzHA for instructions).
>      These may take more time to triage than the other options.

**Comment by aminya** [comment]
Could you check the link again? I removed the not-needed folders so you can just see what is needed.
https://repl.it/@aminya/x-terminal#rollup.config.js

**Comment by cyclops24** [comment]
@aminya @shellscape I'm also using `plugin-multi-entry` and it's working without any issue but It's creating a dummy file in this path: `build/_virtual/_rollup:plugin-multi-entry:entry-point` and because this file name contains `:` in its name, my users on windows can't use my library because `:` character is not allowed in windows OS. We don't have any issues with Linux and Mac.
I think the issue is related to this line:
https://github.com/rollup/plugins/blob/825ef02252cd8fc64adf88473107d9d44404a2fb/packages/multi-entry/src/index.js#L5

Any plan to fix this? I think choosing `:` character as a separator is a bad decision as it's not supported on other OS.

**Comment by frank-dspeed** [comment]
@shellscape watch the comment above me there we have a example of a big where we can not even use the repl to reproduce that as the ``` : ``` char is simply not supported on windows the repl does not give us a option for that we should talk about how to handle that.

**Comment by Bluefinger** [comment]
I can confirm I'm having this same issue with plugin-multi-entry, again on a Windows OS (Win 10). On my private project, I included multi-entry for the intent of forcing certain files to be provided on the bundle entry point, but with my build/watch process, I end up with this error:
```
{
  code: 'ERROR',
  error: [Error: ENOENT: no such file or directory, open 'E:\Code\JSHell\repayne\static\js\_rollup:plugin-multi-entry:entry-point.js.map'] {
    errno: -4058,
    code: 'ENOENT',
    syscall: 'open',
    path: 'E:\\Code\\JSHell\\repayne\\static\\js\\_rollup:plugin-multi-entry:entry-point.js.map'
  }
}
```

- Rollup version: 2.22.2
- Rollup Multi-Entry version: 3.0.1
- Operating System: Windows 10
- Node version: 14.4.0

**Comment by cyclops24** [comment]
Related issue: https://github.com/rollup/plugins/issues/95

**Comment by aminya** [comment]
> @aminya @shellscape I'm also using `plugin-multi-entry` and it's working without any issue but It's creating a dummy file in this path: `build/_virtual/_rollup:plugin-multi-entry:entry-point` and because this file name contains `:` in its name, my users on windows can't use my library because `:` character is not allowed in windows OS. We don't have any issues with Linux and Mac.
> I think the issue is related to this line:
> https://github.com/rollup/plugins/blob/825ef02252cd8fc64adf88473107d9d44404a2fb/packages/multi-entry/src/index.js#L5
> 
> Any plan to fix this? I think choosing `:` character as a separator in the file name is a bad decision as it's not supported on other OS.

Thank you! I am very happy that you found the reason! Now, I am sure they can fix it. 

@frank-dspeed 
You don't have to use Repl for this. There are many number of VM machines available that can help.

**Comment by frank-dspeed** [comment]
@aminya the repl thing is a general thing here it is a internal none related discussion between me and @shellscape :)

**Comment by aminya** [comment]
@frank-dspeed I made a pull request. Check it out.

**Comment by shellscape** [comment]
Fixed by #520

**Comment by cyclops24** [comment]
The multi-entry plugin still create a file with `:` character like this: ` _virtual\_virtual:multi-entry.js`
I described more in the related PR: https://github.com/rollup/plugins/pull/520

@shellscape So please reopen this issue until an actual fix. :pray:

**Comment by shellscape** [comment]
@cyclops24 the right way to track this is with a new issue.

**Comment by aminya** [comment]
In addition to fixing the problem, I added "real-world tests" in https://github.com/rollup/plugins/pull/506/files which could prevent this PR to get merged.

However, due to the behavior of the moderator, I decided to stop using Rollup. To me, a community with a healthy atmosphere is much more important than the actual software. I prefer to contribute to a welcoming community even if the software is the best in the globe.
